web/src/i18n/locales/en/stateDashboard.yaml

```diff
@@ -1,13 +1,14 @@
 title: My State Dashboard
 subheader:
-helpText: >-
-  The CMS HITECH APD app is the new way to create and manage your state’s HITECH
-  Advanced Planning Documents (APDs). Follow the steps to fill in the required information,
-  and the app will generate a completed APD you can submit to a CMS State Officer
-  for review.
+instruction:
```

Comment:
@jameshupp @austinhernandez Changed this to an instruction detail to match the style in #1378.  Is that correct, or should this still be help text, inside the little blue box and everything?

Reply:
No, if this page is styled according to the same rules as the actual app, this is right.